packages/p2p/src/components/modal-manager/modals/market-rate-change-error-modal/market-rate-change-error-modal.jsx

```diff
@@ -4,14 +4,16 @@ import { isMobile } from '@deriv/shared';
 import { localize, Localize } from 'Components/i18next';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
 
-const MarketRateChangeErrorModal = () => {
+const MarketRateChangeErrorModal = ({ message }) => {
     const { is_modal_open, hideModal } = useModalManagerContext();
 
     return (
         <Modal is_open={is_modal_open} onExited={hideModal} small>
             <Modal.Body>
                 <Text as='p' size={isMobile() ? 'xxs' : 'xs'} line_height='s'>
-                    <Localize i18n_default_text={'The advertiser changed the rate before you confirmed the order.'} />
+                    {message ?? (
```

Comment:
Can it be like this?

```
{message ?? (
<Modal.Body>
                <Text as='p' size={isMobile() ? 'xxs' : 'xs'} line_height='s'>
                        <Localize i18n_default_text='The rate of the advert has changed. Please try creating your order again.' />
                </Text>
            </Modal.Body>
)}
```

Reply:
hmmm it can be like that butttt i think its better we leave the message wrapped inside the Text wrapper so it can also adapt to the sizing depending if we're on mobile or not

Reply:
```
{message ?? 
     <Text as='p' size={isMobile() ? 'xxs' : 'xs'} line_height='s'>
                        <Localize i18n_default_text='The rate of the advert has changed. Please try creating your order again.' />
                </Text>
}
```

Reply:
what about this?

Reply:
let me check 👀

Reply:
alright it works 👍